libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2688,7 +2688,28 @@ vector<VariableDeclaration const*> SMTEncoder::stateVariablesIncludingInheritedA
 
 vector<VariableDeclaration const*> SMTEncoder::localVariablesIncludingModifiers(FunctionDefinition const& _function, ContractDefinition const* _contract)
 {
-	return _function.localVariables() + modifiersVariables(_function, _contract);
+	return _function.localVariables() + tryCatchVariables(_function) + modifiersVariables(_function, _contract);
+}
+
+vector<VariableDeclaration const*> SMTEncoder::tryCatchVariables(FunctionDefinition const& _function)
+{
+	struct TryCatchVarsVisitor : public ASTConstVisitor{
+		vector<VariableDeclaration const*> m_vars;
```

Comment:
Please move this to after the function.

Reply:
Also since it's not public please rename it to `vars`.

Reply:
Actually, I made it public (using `struct`, not `class`), since I did not saw the point of writing a getter, as this class is local to the method and thus not visible anywhere else.